localstack/services/cloudformation/engine/template_deployer.py

```diff
@@ -325,7 +327,10 @@ def _resolve_refs_recursively(
             )
             # TODO: we should check the deployment state and not try to GetAtt from a resource that is still IN_PROGRESS or hasn't started yet.
             if resolved_getatt is None:
-                raise DependencyNotYetSatisfied(resource_ids=resource_logical_id, message="")
+                raise DependencyNotYetSatisfied(
+                    resource_ids=resource_logical_id,
+                    message=f"Could not resolve attribute {attribute_name} on resource {resource_logical_id}",
```

Comment:
nit: enclosing replaced values with quotation marks might make this a bit more readable